src/BottomNavigation/BottomNavigation.js

```diff
@@ -0,0 +1,100 @@
+/**
+ * TODOs:
+ * - Support custom highlight colors
+ * - Support custom highlight color per selected item
+ * - Support badged icons
+ * - Support showing label only for selected item
+ * - Add constraints for number of children provided
+ * - Enforce appropriate label length and avoid overflows/breaks
+ * - V2: Support custom background color on bar depending on selected item
+ * - V2: Support expanding width of selected item
+ * - V2: Add scrolling behaviours (hiding)
+ * - V2: Determine how to handle larger displays (menu on left)
+ */
+
+import React from 'react';
+
+function getStyles(props, context) {
+  const {
+    bottomNavigation,
+    zIndex,
+  } = context.muiTheme;
+
+  const styles = {
+    root: {
+      position: 'relative',
+      zIndex: zIndex.bottomNavigation,
+      width: '100%',
+      display: 'flex',
+      justifyContent: 'center',
+      backgroundColor: bottomNavigation.color,
+      height: bottomNavigation.height,
+    },
+    item: {
+      flex: '1',
```

Comment:
We could, but as a general principle, I prefer to have parent elements designate alignment and sizing of children, because I find modularity tends to improve when children don't know about their surroundings (in this case, the child needn't know that its parent is `display: flex`).